--- otp_src_R11B-5/erts/emulator/hipe/hipe_x86_signal.c.orig	2007-11-29 13:34:12.000000000 +0000
+++ otp_src_R11B-5/erts/emulator/hipe/hipe_x86_signal.c	2007-11-29 13:34:21.000000000 +0000
@@ -34,7 +34,7 @@
 #endif
 #include "hipe_signal.h"
 
-#if __GLIBC__ == 2 && (__GLIBC_MINOR__ >= 3 && __GLIBC_MINOR__ <= 6)
+#if __GLIBC__ == 2 && (__GLIBC_MINOR__ >= 3)
 /* See comment below for glibc 2.2. */
 #ifndef __USE_GNU
 #define __USE_GNU		/* to un-hide RTLD_NEXT */
